modules/logging-release-notes-5-8-11.adoc

```diff
@@ -0,0 +1,53 @@
+// module included in /logging/logging-5-8-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-8-11_{context}"]
+= Logging 5.8.11
+This release includes link:https://access.redhat.com/errata/RHBA-2024:5124[OpenShift Logging Bug Fix Release 5.8.11] and link:https://access.redhat.com/errata/RHBA-2024:5123[OpenShift Logging Bug Fix Release 5.8.11].
+
+[id="logging-release-notes-5-8-11-bug-fixes"]
+== Bug fixes
+* Before this update, the TLS section was added without verifying the broker URL schema, leading to SSL connection errors if the URLs didn't start with `tls`. With this update, the TLS section is added only if broker URLs start with `tls`, preventing SSL connection errors. (link:https://issues.redhat.com/browse/LOG-5139[LOG-5139])
+
+* Before this update, the Loki Operator did not trigger alerts when it dropped log events due to validation failures. With this update, the Loki Operator includes a new alert definition that triggers an alert if Loki drops log events due to validation failures. (link:https://issues.redhat.com/browse/LOG-5896[LOG-5896])
```

Comment:
Not sure if it's appropriate to refer to "Loki" anthropomorphically, or if it should always be "Loki Operator".

Reply:
Only using "Loki" instead of "Loki Operator" in the second part of the sentence is intentional. The operator is not dropping log events, Loki does that.